src/simulation/power/simpleBattery/simpleBattery.cpp

```diff
@@ -59,6 +59,16 @@ void SimpleBattery::evaluateBatteryModel(PowerStorageStatusMsgPayload *msg) {
         this->storedCharge = 0;
     }
 
+    if(this->fault == true) {   
```

Comment:
Actually, is there a reason to have both `fault` AND `faultCapacityRatio`? Doesn’t `faultCapacityRatio==1` imply no fault in the capacity and anything below 1 implies a fault?

Reply:
I would also do `if(this->fault)` instead of `if(this->fault == true)`.

Reply:
I agree. If the fault info is coming from an optional input message, we don't need `fault` at all.  Rather, if a fault message is connected, then the fault is read in and applied.  If we temporarily don't want a fault, then the message value of `faultCapacityRatio` would just be set to 1?